scaffolding-chef-inspec/lib/scaffolding.ps1

```diff
@@ -0,0 +1,176 @@
+#
+# A scaffolding for an InSpec profile
+#
+
+function Load-Scaffolding {
+    $pkg_deps += @(
+        "${pkg_deps[@]}"
+        "stuartpreston/inspec"
+    )
+    $pkg_build_deps += @(
+        "${pkg_build_deps[@]}"
+        "stuartpreston/inspec"
+    )
+
+    $pkg_svc_user="administrator"
+    $pkg_svc_run = "set_just_so_you_will_render"
+}
+
+function Invoke-DefaultBefore {
+    if(!(Test-Path "$PLAN_CONTEXT/../inspec.yml")) {
+        Write-BuildLine "Error: Cannot find inspec.yml"
+        Write-BuildLine "  Please build from the profile root"
+        exit 1
+    }
+
+    if((Select-String -Path "$PLAN_CONTEXT/../inspec.yml" -Pattern "compliance: ")){ 
+        Invoke-ComplianceLogin
+    }
+}
+
+function Invoke-DefaultUnpack {
+    Copy-Item "$PLAN_CONTEXT/../" "$HAB_CACHE_SRC_PATH/$pkg_dirname" -Exclude "habitat", "results", "*.harts"
+}
```

Comment:
`Copy-Item` is creating that directory structure but instead of using `ls` to build out a list then copy them individually the `-Exclude` is just leaving out the unnecessary habitat files.

Reply:
PERFECT!